vscode-lean4/src/config.ts

```diff
@@ -162,6 +162,10 @@ export function serverLoggingPath(): string {
     return workspace.getConfiguration('lean4.serverLogging').get('path', '.')
 }
 
+export function autofocusOutput(): boolean {
```

Comment:
```suggestion
export function shouldAutofocusOutput(): boolean {
```

Reply:
(Since otherwise it sounds like something will be autofocused when we call this.)